app/plugin_install.go

```diff
@@ -45,6 +51,10 @@ func (a *App) RemovePluginFromData(data model.PluginEventData) {
 	if err := a.removePluginLocally(data.Id); err != nil {
 		mlog.Error("Failed to remove plugin locally", mlog.Err(err), mlog.String("id", data.Id))
 	}
+
+	if err := a.notifyPluginStatusesChanged(); err != nil {
```

Comment:
Just looking at the diff at this point in the review, and it's entirely not clear why is a `notifyPluginEvents` and a `notifyPluginStatusesChanged` -- feels like we could converge on fewer methods?

Reply:
True, it's just a helper to combine `notifyPluginStatusesChanged` and `notifyPluginEnabled`. Got rid of it, moved the `IsActive` inside `notifyPluginEnabled`.